build/k8s/ingestor.yaml

```diff
@@ -108,10 +109,20 @@ spec:
         adx-mon/log-destination: "Logs:Ingestor"
         adx-mon/log-parsers: json
     spec:
+      automountServiceAccountToken: true
+      securityContext:
+        fsGroup: 65534
```

Comment:
I think we should remove the `fsGroup` properties from the two ingestor configs - we're already running as the `root` user to be able to read/write our own WAL files, so setting the group to the nobody group isn't really doing anything.